scenarios/iam_privesc_by_attachment/cheat_sheet_kerrigan.md

```diff
@@ -18,10 +18,6 @@
 
 `aws ec2 run-instances --image-id ami-0a313d6098716f372 --iam-instance-profile Arn=<instanceProfileArn> --key-name kerrigan --profile kerrigan --subnet-id <subnetId> --security-group-ids <securityGroupId>`
 
-`sudo apt-get update`
-
-`sudo apt-get install awscli`
-
```

Comment:
This is going to apply to every scenario and will differ based on the OS used. Thinking there's likely a better place to put this (but not sure where that might be at the moment).

Reply:
I think the reason I deleted this is because the `aws` utility is used in many lines above this, so there is no need to install the utility at this point in the solution because you've already used it. Maybe the following improvements would be better:
1. Place aws CLI install at the top of each solution and use the `curl` installation method [outlined here](https://docs.aws.amazon.com/cli/latest/userguide/install-linux.html#install-linux-bundled-sudo) rather than apt (which is Ubuntu-specific)
2. Replace an installation command with a small note about install AWS CLI with a link to the OS-agnostic AWS v1 CLI install page [here](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html)

Reply:
Ah... this is removing those lines, not adding them. Not sure how I mixed that up, this makes sense.

Saw you closed this PR. I'll probably go through all the cheat sheets some time soon and make sure they make sense. Can come back to these changes then I think.